docs/server-side/v2/changelog/release-notes.mdx

```diff
@@ -7,9 +7,9 @@ This Changelog highlights notable changes to the 100ms server-side API, such as
 
 ## 2023-10-17
 - Added support for RTMP ingestion to live stream on 100ms from a RTMP broadcaster. Added documentation for:
-    - [Create RTMP stream key for a specific room](/server-side/v2/api-reference/live-streams/create-rtmp-stream-key)
-    - [Get RTMP stream key and URL for a specific room](/server-side/v2/api-reference/live-streams/get-rtmp-stream-key)
-    - [Disable RTMP stream key for a specific room](/server-side/v2/api-reference/live-streams/disable-rtmp-stream-key)
+    - [Create RTMP stream key for a specific room](/server-side/v2/api-reference/stream-key/create-rtmp-stream-key)
```

Comment:
include the new field in hlsDestinations object in policyAPI - we are surfacing this to configure the beam autostop Timeout on a destination level

Reply:
@vivekd-100ms will push a PR separately along with release notes for this.
Will include when i create the how-to doc.